jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql5-cluster-jbpm-schema.sql

```diff
@@ -0,0 +1,854 @@
+    create table Attachment (
+        id bigint not null auto_increment,
+        accessType integer,
+        attachedAt datetime(6),
+        attachmentContentId bigint not null,
+        contentType varchar(255),
+        name varchar(255),
+        attachment_size integer,
+        attachedBy_id varchar(255),
+        TaskData_Attachments_Id bigint,
+        primary key (id)
+    );
```

Comment:
you're missing the `ENGINE=InnoDB;` after each `create table` statement. Please add it to all

Reply:
added